docs/moactl_create_cluster.md

```diff
@@ -28,8 +28,7 @@ moactl create cluster [flags]
       --version string                Version of OpenShift that will be used to install the cluster, for example "4.3.10"
       --multi-az                      Deploy to multiple data centers.
       --compute-machine-type string   Instance type for the compute nodes. Determines the amount of memory and vCPU allocated to each compute node.
-      --compute-nodes int             Number of worker nodes to provision per zone. Single zone clusters need at least 4 nodes, while multizone clusters need at least 9 nodes (3 per zone) for resiliency.
-                                      
+      --compute-nodes int             Number of worker nodes to provision per zone. Single zone clusters need at least 4 nodes, multizone clusters need at least 9 nodes. (default 4)
```

Comment:
Maybe should say (default 4 or 9, respectively) or something like that.

Reply:
That's a tricky one, as the `(default 4)` is added automatically by the cobra library... the alternative would be to _not_ set a default (so nothing shows up in the help text) but then force a default value in the code...